proto/buf/alpha/registry/v1alpha1/jsonschema.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+// JSONSchemaService serves JSONSchemas describing protobuf types in buf
+// modules.
+service JSONSchemaService {
+  // GetJSONSchema allows users to get an (approximate) json schema for a
+  // protobuf type.
+  rpc GetJSONSchema(GetJSONSchemaRequest) returns (GetJSONSchemaResponse);
+}
+
+
+message GetJSONSchemaRequest {
+  string owner = 1;
+  string repository = 2;
+  string reference = 3;
+
+  // A fully qualified name of the type to generate a JSONSchema for, e.g.
+  // "pkg.foo.Bar".
```

Comment:
Given https://github.com/bufbuild/buf/pull/885#discussion_r792040774, should we call out that the leading `.` is dropped or is this good enough with the example?